src/en/wizden-staff/maintainer/review-procedure.md

```diff
@@ -1,54 +1,181 @@
 
-- Any PR that is made for illegitimate reasons, such as abuse, spam or harrassment, may be closed without following the Policy as long as a PM has given written permission, as per the procedures set out in the Requirements section.
 
-- Not following the procedure/policy outlined in this document will result in disiplinary action being taken, at the discretion of the SS14 Wizard team.
+- All outstanding maintainer change requests must be resolved.
+If the maintainer who left the request wants, they can choose to forgo requiring the changes.
+In this case, when the PR is merged, a Github issue should be created to indicate the changes that need to be made.
 
-## Issues & Reviews
-While Issues do not have any code to review, they may still contain suggestions and feature proposals that would benefit from a Maintainer Discussion.
 
-- An Issue can be given the `S: Undergoing Maintainer Discussion` label to create a discussion thread as per the process described in the Policy section. If the discussion has a positive outcome, the issue can be given the `S: Conceptual Approval` label, and any PR that references the issue without major deviation can subsequently also be given the same label, indicating no new discussion is necessary.
+- If the PR has the `Changes: Sprites` label, it must additionally be approved by at least one **Art Lead** through the addition of the `S: Art Approval` tag.
+Changes that are solely minor sprite fixes are exempt from this.
+
+To close a PR, at **least one** of the following conditions must be met:
+- At least 2 maintainers must express wanting to close the PR or an unwillingness to merge it. 
+This can be expressed either on the PR itself, or privately.
+
+
+- The PR contains content which violates Wizard Den's rules and/or code of conduct.
+
+
+- The PR author has been **banned** from the Wizard's Den Github and/or Discord or is otherwise unable to be communicated with.
+
+
+- The PR is **Stale**.
+
+
+- The PR has been superseded by a different PR and thus no longer needs to be merged.
+
+If a PR is closed, the maintainer must leave a message indicating the reason for its closure.
+This should be a formal message that includes all relevant information.
+If a maintainer discussion was held, it should be summarized and included here. 
+
+If there is disagreement between two maintainers about whether a PR should be merged or closed, a discussion should be held as described in the "Discussion Policy" section.
+
+## Exemptions:
+
+```admonish info "Use Your Head"
+The following are situations in which the typical decision policy can be modified or opted out of.
+If you do not feel confident, feel free to follow the normal process.
+```
+
+### Fixes
+
+PRs which fix bugs, cleanup code, improve performance, or refactor systems _without_ the introduction of new features or balance changes can be processed by a single maintainer (who is not the PR author).
```

Comment:
This could mention the Triage Area tags we have.

Reply:
The previous version of this section mentioned the tags specifically. I choose to omit them in the rewrite because i feel like this wording is more inclusive of both traiged and un-triaged PRs.

Reply:
fair enough